spec/fixtures/middleware/internal_info_setter.rb

```diff
@@ -0,0 +1,11 @@
+class InternalInfoSetter
+  MESSAGE = "set by internal_middleware"
+  def initialize(bugsnag)
+    @bugsnag = bugsnag
+  end
+
+  def call(notification)
+    notification.info = MESSAGE
```

Comment:
I cant see how this works? Is this some ruby awesomeness I dont understand? How does this end up in metadata?